jwst/assign_wcs/miri.py

```diff
@@ -603,3 +603,32 @@ def get_wavelength_range(input_model, path=None):
     band = input_model.meta.instrument.band
 
     return dict([(ch + band, wr[ch + band]) for ch in channel])
+
+
+def store_dithered_position(input_model):
+    """Store the location of the dithered pointing
+    location in the wcsinfo
```

Comment:
Ah, and it's stored in the `dither` section now, not `wcsinfo`.

Reply:
✅